minio/base/externalsecret-minio-admin-credentials.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: minio-admin-credentials
+  namespace: minio
+spec:
+  refreshInterval: 15s
```

Comment:
This seems like an aggressive refresh interval.

Reply:
Sounds good, I made this change.